src/util/favorites.js

```diff
@@ -231,7 +230,14 @@ export const translateConfig = config => {
             return {
                 el,
                 name,
-                alerts: [createAlert(name, i18n.t('Map could not be created'))],
+                alerts: [
+                    {
+                        critical: true,
+                        message: `${name}: ${i18n.t(
+                            'Map could not be created'
```

Comment:
I'm not sure on the context here, but is this for saving a Map? If so, I suggest the text: "There was a problem saving this Map. Changes have not been saved."

Reply:
@cooper-joe, the message is shown if it was not possible to create a map from a chart or pivot.

Reply:
Oh right, thanks for clarifying. Then I think "There was a problem creating a map from the imported data." works well.

Reply:
Fixed.